### PR TITLE
Support 'VALUE_CHANGED'

### PR DESCRIPTION
**Pull request by cshcomcom**

### CODE REVIEW

File: src/event-manager/event-callback-manager.ts [hunk 1]
```diff
@@ -38,22 +38,18 @@ export default class EventCallbackManager {
...
-      case BlockchainEventTypes.VALUE_CHANGED: // TODO(cshcomcom): Implement
-        throw Error(`Not implemented`);
-      case BlockchainEventTypes.TX_STATE_CHANGED: // TODO(cshcomcom): Implement
-        throw Error(`Not implemented`);
-      default:
-        throw Error(`Invalid event type (${eventType})`);
+    if (!Object.values(BlockchainEventTypes).includes(eventType)) {
+      throw Error(`Invalid event type (${eventType})`);
+    } else if (eventType === BlockchainEventTypes.TX_STATE_CHANGED) {
+      throw Error(`Not implemented`); // TODO(isak): Implement
```
Comment (platfowner) [inline src/event-manager/event-callback-manager.ts:44]: minor) period (.) at the end?

File: src/event-manager/event-callback-manager.ts [hunk 2]
```diff
@@ -38,22 +38,18 @@ export default class EventCallbackManager {
...
     }
+    const filterId = this.buildFilterId();
+    if (this._filters.get(filterId)) { // TODO(cshcomcom): Retry logic
```
Comment (platfowner) [inline src/event-manager/event-callback-manager.ts:47]: ditto

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: src/event-manager/event-callback-manager.ts [hunk 3]
```diff
@@ -38,22 +38,18 @@ export default class EventCallbackManager {
...
-        const filter = new EventFilter(filterId, eventType, config);
-        this._filters.set(filterId, filter);
-        return filter;
-      case BlockchainEventTypes.VALUE_CHANGED: // TODO(cshcomcom): Implement
-        throw Error(`Not implemented`);
-      case BlockchainEventTypes.TX_STATE_CHANGED: // TODO(cshcomcom): Implement
-        throw Error(`Not implemented`);
-      default:
-        throw Error(`Invalid event type (${eventType})`);
+    if (!Object.values(BlockchainEventTypes).includes(eventType)) {
```
Comment (liayoo) [inline src/event-manager/event-callback-manager.ts:41]: 👍

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: src/event-manager/event-callback-manager.ts (hunk 1 above)
Comment (cshcomcom) [reply src/event-manager/event-callback-manager.ts]: Done!

File: src/event-manager/event-callback-manager.ts (hunk 2 above)
Comment (cshcomcom) [reply src/event-manager/event-callback-manager.ts]: Done!

**Comment by cshcomcom** [comment]
Thanks for reviews! Since this PR is related with [ain-blockchain PR](https://github.com/ainblockchain/ain-blockchain/pull/933), I will merge them together.

**Pull request merged**